oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentNodeStoreBuilderTest.java

```diff
@@ -60,6 +62,22 @@ public void fullGCFeatureToggleDisabled() {
         assertNull(builder.getDocStoreFullGCFeature());
     }
 
+    @Test
```

Comment:
Please consider unit testing for invalid paths - maybe use a RegEx or a Utils method that validates the path string.

Reply:
added in https://github.com/apache/jackrabbit-oak/pull/1551/commits/167b46c5eaa518d39bad952853ac55bed2e0b7a5